modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -427,6 +429,32 @@ void ONNXImporter::addLayer(LayerParams& layerParams,
     }
 }
 
+void ONNXImporter::expandMid(const LayerParams& layerParams, opencv_onnx::NodeProto& node_proto,
+                             const std::string& input, size_t n, int axis)
+{
+    String base_name = layerParams.name + "/copy_";
+    std::vector<std::string> input_names;
+    for (int j = 0; j < n; j++)
+    {
+        std::ostringstream ss;
+        ss << j;
+        LayerParams copyLP;
+        copyLP.name = base_name + ss.str();
+        copyLP.type = "Identity";
+        CV_Assert(layer_id.find(copyLP.name) == layer_id.end());
+        input_names.push_back(copyLP.name);
+
+        node_proto.set_input(0, input);
+        node_proto.set_output(0, copyLP.name);
+        addLayer(copyLP, node_proto);
+    }
+    node_proto.clear_input();
+    for (int i = 0; i < input_names.size(); i++)
```

Comment:
signed - unsigned comparison - prefer to use `size_t i`

Reply:
done